modules/material-management/test/cryptographic_material.test.ts

```diff
@@ -127,25 +103,18 @@ describe('decorateCryptographicMaterial', () => {
     const suite = new NodeAlgorithmSuite(
       AlgorithmSuiteIdentifier.ALG_AES128_GCM_IV12_TAG16
     )
-    const test = decorateCryptographicMaterial(
-      { suite, keyringTrace: [] } as any,
-      KeyringTraceFlag.WRAPPING_KEY_GENERATED_DATA_KEY
-    )
+    const test = decorateCryptographicMaterial({ suite } as any)
     const dataKey = new Uint8Array(suite.keyLengthBytes).fill(1)
     test.setUnencryptedDataKey(new Uint8Array(dataKey), {
       keyNamespace: 'k',
       keyName: 'k',
-      flags: KeyringTraceFlag.WRAPPING_KEY_GENERATED_DATA_KEY,
     })
```

Comment:
Same as above: shouldn't this keyring trace (second argument) be deleted entirely?